src/SFA.DAS.EAS.Portal.Client/IPortalClient.cs

```diff
@@ -6,6 +6,7 @@ namespace SFA.DAS.EAS.Portal.Client
 {
     public interface IPortalClient
     {
-        Task<Account> GetAccount(long accountId, CancellationToken cancellationToken = default);
+        Task<Account> GetAccount(long accountId, string publicHashedAccountId,
+            bool hasPayeScheme, CancellationToken cancellationToken = default);
```

Comment:
I think we talked about this point earlier on. Creating a class of filters with a HasPayeScheme boolean property. This will allow it to be easily extended if more flags are added in the future.

Also could we not has the id in the client rather than passing it in alongside the accountId ?

Reply:
I've changed the signature to accept a flags enum, so that we if we need to pass more flags, we don't have to add further parameters. It also has the bonus of making the consumer code more readable, because it will pass `AccountState.HasPayeScheme` rather than `true`.

I've also moved decoding of the hashed `AccountId` into the client, so now the method only has to accept a hashed `AccountId`, rather than the hashed and vanilla versions.

Reply:
👍  on the hashing, I think there are comments further down to potentially argue that a simple class with different flags on might be more appropriate, if we have IncludeVacancyData = true AND MaxVacancies = 2

Reply:
The correct location of the client related logic is being debated in other threads, and the outcome of those threads will inform the interface.

Reply:
I've now introduced a (business) parameter object for the `GetAccount` method, as agreed offline.